franklin/builder/models.py

```diff
@@ -26,3 +46,8 @@ def save(self, *args, **kwargs):
             os.environ['BASE_URL']
         )
         super(Site, self).save(*args, **kwargs)
+        self.build(
+            git_hash=self.git_hash,
+            repo_owner='istrategylabs',
+            path=self.path,
+            repo_name=self.repo_name)
```

Comment:
Totally right, I started writing this as it would be separate task and forgot to update this.